templates/partial/_navigation.html

```diff
@@ -1,37 +1,91 @@
 <a href="#main-content" class="p-link--skip">Jump to main content</a>
 
-<header id="navigation" class="p-navigation{%if not request.path|get_nav_path == 'partners'%} is-sticky{% endif %}">
+<header id="navigation" class="p-navigation {%if not request.path|get_nav_path == 'partners'%}is-sticky{% endif %} is-dark">
   {% block header_banner %}
   <div class="p-navigation__row">
     <div class="p-navigation__banner">
-      <div class="p-navigation__logo">
-        <a class="p-navigation__item" href="/">
-          <img class="p-navigation__image" src="https://assets.ubuntu.com/v1/2eec3fd9-logo.svg" alt="" width="115">
-          <script>performance.mark("Logo rendered")</script>
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="/">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="">
+          </div>
+          <span class="p-navigation__logo-title">Canonical</span>
         </a>
       </div>
       <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
       <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
     </div>
-    <nav class="p-navigation__nav">
+
+    <nav class="p-navigation__nav" aria-label="main-navigation">
       <ul class="p-navigation__items">
-        <li
-          class="p-navigation__item {% if request.path|get_nav_path == 'products' %}is-selected{% endif %}"
-          role="menuitem">
-          <a class="p-navigation__link" href="/products">Products</a>
+        <li class="p-navigation__item--dropdown-toggle" id="company">
```

Comment:
@spencerbygraves @lyubomir-popov @al-bakalova - can we get some advice on this please - if a navigation item is within a dropdown menu, how do we indicate the currently active nav item to the user?

![Peek 2022-10-14 11-48](https://user-images.githubusercontent.com/2376968/195829682-7c4c5c3a-ece5-4e98-b9de-fb529ea05a5a.gif)

My thought is to add `.is-selected` to the dropdown toggle, and then an active state to the links (which I'm not sure exists in Vanilla). What do you think?

Reply:
Demo here: https://canonical-com-650.demos.haus/